packages/block-library/src/navigation/edit/index.js

```diff
@@ -92,7 +92,7 @@ function Navigation( {
 			flexWrap = 'wrap',
 		} = {},
 		hasIcon,
-		icon,
+		icon = 'handle',
```

Comment:
The block does have a default attribute of 'handle', so this may not be required, but I don't mind a bit of extra safety.